sdks/java/extensions/schemaio-expansion-service/build.gradle

```
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id 'org.apache.beam.module'
+}
+
+applyJavaNature(
+        ignoreRawtypeErrors: true,
+        automaticModuleName: 'org.apache.beam.sdk.extensions.schemaio.expansion',
+        validateShadowJar: false,
+        shadowClosure: {
+            manifest {
+                attributes "Main-Class": "org.apache.beam.sdk.expansion.service.ExpansionService"
+            }
+        }
+)
+
+dependencies {
+    compile project(path: ":sdks:java:expansion-service")
+    compile project(":sdks:java:io:jdbc")
+    compile library.java.postgres
+    testCompile library.java.junit
+    testCompile library.java.powermock_mockito
+    testCompile library.java.mockito_core
+    testCompile "org.checkerframework:checker-qual:3.5.0"
```

Assuming this is because of BEAM-10632? Let's add a TODO here so we remember to remove it later
```suggestion
    testCompile library.java.mockito_core
    // TODO(BEAM-10632): remove this dependency
    testCompile "org.checkerframework:checker-qual:3.5.0"
```

---

Done.